src/ImageUtility.ps1

```diff
@@ -0,0 +1,38 @@
+﻿function Export-Entity([string] $entityPath, [string[]] $fileNames)
+{
+    $srcDir = [io.path]::combine($Config.AuditDir, $entityPath)
+    $dstDir = [io.path]::combine($Config.ArtifactsDir, $entityPath)
+
+    New-Item -Path $dstDir -ItemType directory -Force
+
+    Get-ChildItem -path $srcDir | ForEach-Object {
+        $dbImageDir = [io.path]::combine($srcDir, $_.Name)
+
+        $fileNames | ForEach-Object {
+            Export-File -srcDir $dbImageDir -fileName $_ -dstDir $dstDir

```

Comment:
You will not get unique names for files. All your files will get the name "avatar.jpg" or "avatar.small.jpg". Is that really what you want?

Reply:
No, file names will be unique, e.g. Adam-Sitnik.avatar.jpg